lib/svg.dart

```diff
@@ -1,51 +1,42 @@
 import 'dart:ui';
+import 'package:flame/assets/assets_cache.dart';
+import 'package:flame/extensions/vector2.dart';
+import 'package:flame/game/game.dart';
 import 'package:flutter_svg/flutter_svg.dart';
 
 import 'package:flame/flame.dart';
-import 'package:flame/position.dart';
 
 class Svg {
   DrawableRoot svgRoot;
   Size size;
 
-  Svg(String fileName) {
-    Flame.assets.readFile(fileName).then((svgString) async {
-      svgRoot = await svg.fromSvgString(svgString, svgString);
-    });
-  }
+  Svg(this.svgRoot);
 
-  /// Renders the svg on the [canvas] using the dimensions provided on [width] and [height]
-  ///
-  /// If not loaded, does nothing
-  void render(Canvas canvas, double width, double height) {
-    if (!loaded()) {
-      return;
-    }
+  static Future<Svg> load(String fileName, {AssetsCache cache}) async {
```

Comment:
Nit: The default in the parameter don't work here? like

```dart
  static Future<Svg> load(String fileName, {AssetsCache cache = Flame.assets}) async {
```

Reply:
Unfortunately not, default parameters only work with constants.

Reply:
point to JS hahaha